app/views/devise/sessions/new.html.erb

```diff
@@ -34,11 +34,14 @@
       ) %>
   <%= f.input :request_id, as: :hidden, input_html: { value: @request_id } %>
   <div class='margin-bottom-4'>
-    <%= f.submit t('links.next'), full_width: true, big: false, wide: false, class: 'margin-bottom-2' %>
+    <%= f.submit t('links.next'), full_width: true, big: false, wide: false, class: 'margin-bottom-3' %>
+    <h2 class='margin-top-1 margin-bottom-3 text-normal text-center text-center--line text-bold border-bottom'>
+      <span class='padding-left-1 padding-right-1 bg-white' ><%= t('headings.create_account_with_sp.cta', app_name: APP_NAME) %></span>
```

Comment:
[maybe we could use the `-x-` styles](https://designsystem.digital.gov/utilities/margin-and-padding/#:~:text=Margin%20left%20and%20right) for left + right?
```suggestion
      <span class='padding-x-1 bg-white' ><%= t('headings.create_account_with_sp.cta', app_name: APP_NAME) %></span>
```

Reply:
yeah this is a lot better. thanks!